generators/python/sdk/CHANGELOG.md

```diff
@@ -5,6 +5,45 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.11.1] - 2024-02-20
+
+- Improvement: Python now supports specifying files to auto-export from the root `__init__.py` file, this means you can export custom classes and functions from your package for users to access like so:
+
+  ```python
+  from my_package import custom_function
+  ```
+
+  the configuration for this is:
+
+  ```yaml
+  # generators.yml
+  python-sdk:
+    generators:
+      - name: fernapi/fern-python-sdk
+        version: 0.11.1
+        config:
+          additional_init_exports:
+            - from: file_with_custom_function
+              imports:
+                - custom_function
+  ```
+
+- Chore: Add a docstring for base clients to explain usage, example:
+
+  ```python
+  class SeedTest:
+    """
+    Use this class to access the different functions within the SDK. You can instantiate any number of clients with different configuration that will propogate to these functions.
+    ---
+    from seed.client import SeedTest
+
+    client = SeedTest(
+        token="YOUR_TOKEN",
+        base_url="https://yourhost.com/path/to/api",
+    )
+    """
```

Comment:
do you think it's easy to add the parameter level documentation as well just like we do for endpoints? 

```python
token: Optional[str]. The token. 

base_url: Optional[str]. The Sever URL used by the SDK. 
```

Reply:
Let me check!

Reply:
Updated!